neuralmonkey/runners/logits_runner.py

```diff
@@ -152,6 +157,7 @@ def get_executable(self,
         fetches["logits"] = self._decoder_logits
 
         return LogitsExecutable(self.all_coders, fetches,
+                                num_sessions,
```

Comment:
zaříznout za self.all_coders a zvážit zaříznout za `(` a odsadit blíž ke zdi

Reply:
DONE